.gitignore

```diff
@@ -7,11 +7,11 @@ crypto-ffi/bindings/*/out
 crypto-ffi/bindings/*/.build
 crypto-ffi/bindings/*/build
 crypto-ffi/bindings/*/.swiftpm
-crypto-ffi/bindings/swift/WireCoreCrypto/WireCoreCrypto/CoreCrypto.swift
-crypto-ffi/bindings/swift/WireCoreCrypto/WireCoreCrypto/core_crypto.swift
-crypto-ffi/bindings/swift/WireCoreCrypto/WireCoreCrypto/core_cryptoFFI.h
-crypto-ffi/bindings/swift/WireCoreCrypto/WireCoreCrypto/core_crypto_ffiFFI.h
-crypto-ffi/bindings/swift/WireCoreCrypto/WireCoreCrypto/core_cryptoFFI.modulemap
+crypto-ffi/bindings/swift/WireCoreCryptoUniffi/WireCoreCryptoUniffi/core_crypto_ffi.swift
+crypto-ffi/bindings/swift/WireCoreCryptoUniffi/WireCoreCryptoUniffi/core_crypto.swift
+crypto-ffi/bindings/swift/WireCoreCryptoUniffi/WireCoreCryptoUniffi/core_cryptoFFI.h
+crypto-ffi/bindings/swift/WireCoreCryptoUniffi/WireCoreCryptoUniffi/core_crypto_ffiFFI.h
+crypto-ffi/bindings/swift/WireCoreCryptoUniffi/WireCoreCryptoUniffi/core_cryptoFFI.modulemap
```

Comment:
I think we should move this out of the root `.gitignore` and instead have one in `crypto-ffi/bindings/swift/WireCoreCryptoUniffi/WireCoreCryptoUniffi` that ignores everything.